source/docs/articles/drupal/major-version-drupal-upgrades.md

```diff
@@ -6,26 +6,42 @@ category:
   - managing
 keywords: drupal, upgrade, upgrading, revision upgrade, update, updating
 ---
-The best practice for doing a major Drupal revision upgrade (e.g. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
+The best practice for doing a major Drupal revision upgrade (i.e. version 6 to version 7) is to start a new site. Even the simplest of upgrades require their own QA and deployment process, and trying to do an upgrade on an existing site is not a recipe for success.
```

Comment:
It's the only way, not just a best practice. `To upgrade Drupal to a new major version...`